data/recipe.txt

```diff
@@ -0,0 +1,9 @@
+pizza | 34 mins | 340 kcals | eggs | DINNER | 2024-04-03 | dinner
+pizza | 34 mins | 340 kcals | eggs | DINNER | 2024-04-03 | dinner
+pizza | 34 mins | 340 kcals | eggs | DINNER | 2024-04-03 | dinner
+pizza | 34 mins | 340 kcals | eggs | DINNER | 2024-04-03 | dinner
+name | 34 mins | 340 kcals | allergies | DINNER | 2024-04-03 | dinner
+newrecipe | 34 mins | 340 kcals | egg | DINNER | 2024-04-03 | dinner
+pizza | 34 mins | 340 kcals | egg | DINNER | 2024-04-03 | www.food.com
+pizza | 34 mins | 340 kcals | allergy | DINNER | 2024-04-03 | url
+pizza | 34 mins | 340 kcals | egg,nut | DINNER | 2024-04-03 | url
```

Comment:
Hi Nidhi, let's discuss again about having a default list of recipes. Let's not push this for now.